webapp/components/create_post.jsx

```diff
@@ -92,8 +93,12 @@ export default class CreatePost extends React.Component {
             return;
         }
 
-        if (post.message.length > Constants.CHARACTER_LIMIT) {
-            this.setState({postError: `Post length must be less than ${Constants.CHARACTER_LIMIT} characters.`});
+        if (this.state.postError) {
+            const errorElement = $('.post-error');
+            errorElement.addClass('animation--highlight');
+            setTimeout(() => {
+                errorElement.removeClass('animation--highlight');
+            }, 1000);
```

Comment:
Can you remove jquery and do this by setting state (for example, `this.setState({errorClass: 'animation--highlight'})`? All UI should be handled by the render method and we're also trying to remove jquery from the codebase since it's such a large library and it makes it too easy to bypass the proper React flow